scripts/evaluator.py

```diff
@@ -0,0 +1,338 @@
+import argparse
+import csv
+import json
+import os
+import warnings
+
+import cv2
+import numpy as np
+import torch
+from tqdm import tqdm
+
+from scripts.goggle_classifier import get_model
+from src.jetson.face_detector import FaceDetector
+from src.jetson.classifier import Classifier
+from scripts.utils import check_rotation, correct_rotation, bbox_iou
+
+PRED_DETECTIONS_FILE = 'detection_predictions.csv'
+CLASSIFICATION_RESULTS_FILE = 'results.json'
+VIDEO_EXT = ['.mov', '.mp4', '.avi', '.MOV', '.MP4', '.AVI']
+
+"""
+Evaluate classification and (optionally) face detection ability on a set of videos.
+Videos to be evaluated should be from the TestVideos folder on the Drive.
+To compare face detection models, run annotator.py first.
+"""
+
+
+class Evaluator():
```

Comment:
this might be better suited in its own file. I know this is the only place its used but this is excessive for a script. I know I previously mentioned its okay to define script specific code in the script file but that should be smaller helper functions not large classes and data models

Reply:
I adopted this script from Aditya, and he had everything here as a class. Should I still leave it as a class?

Reply:
yes leaving this as a class is fine. just move it out to its own file. This will help make the next task refactoring much easier.